corehq/apps/users/dbaccessors.py

```diff
@@ -183,6 +191,11 @@ def _get_invitations_by_filters(domain, user_filters, count_only=False):
     support ES search syntax, it's just a case-insensitive substring search.
     Ignores any other filters.
     """
+    only_active = user_filters.get("user_active_status", None)
+    if not only_active and only_active is not None:
```

Comment:
Is this conditional right? I'm interpreting `user_active_status=True` means returning only invitations whose users are active on the domain (which would be 0). But that conditional would evaluate as False